### DISCUSSION TITLE
[@types/luxon] Property 'ts' does not exist on type 'DateTime<true>'.

### DISCUSSION THREAD

**Discussion by alix1383** [DefinitelyTyped/DefinitelyTyped#68191] [Issues with a @types package] [answered]
[{
	"owner": "_generated_diagnostic_collection_name_#1",
	"message": "Property 'ts' does not exist on type 'DateTime<true>'.",
        "code" : "DateTime.now().ts",
	"source": "ts",
}]

[a comment by an automated account is not shown]

**Comment by Semigradsky** [comment]
@alix1383 Could you please provide a link to documentation where `ts` described?

**Comment by CarsonF** [comment, marked as answer]
We've never exposed the `ts` property, it's an implementation detail.
https://github.com/moment/luxon/blob/master/src/datetime.js#L477-L480

**Comment by alix1383** [comment]
I see, sorry for wrong question
